lib/reconcile.js

```diff
@@ -197,6 +209,8 @@ module.exports = function (clusterInfo, uuid = require('uuid')) {
 
     const { environment = 'production' } = cluster
 
+    delete cluster.operation
```

Comment:
unrelated change, but should solve that weird error we saw where it reported a deleted cluster as still existing